src/System.Security.Cryptography.Cng/ref/project.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.IO": "4.0.0",
-    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime": "4.1.0-rc3-24117-00",
```

Comment:
Also, while I know the data is in the diag log, I was looking to try to eliminate these myself the other day, but couldn't find a succinct place where it says what's wrong.  Is it the thinking I think it is, or is there a simple place that I missed?